src/ol/featureloader.js

```diff
@@ -87,8 +87,10 @@ ol.featureloader.tile = function(url, format) {
        * @this {ol.VectorTile}
        */
       function(features, dataProjection) {
-        this.setProjection(dataProjection);
-        this.setFeatures(features);
+        /** @type {ol.VectorTile} */
+        var me = this;
```

Comment:
I suspect That he takes this definition: https://github.com/sbrunner/ol3/blob/95d30c1cbf592c49b67ff4247dda8b5fae43c076/src/ol/featureloader.js#L12

The `setState` from the function bellow, is also available in the `Source`...